CMakeLists.txt

```diff
@@ -500,23 +505,23 @@ IF(ENABLE_TESTING)
 		ENDIF()
 
 		IF(BUILD_META_EXAMPLES AND NOT DISABLE_META_INTEGRATION_TESTS AND EXISTS ${CMAKE_SOURCE_DIR}/tests/meta)
-            add_subdirectory(${CMAKE_SOURCE_DIR}/tests/meta)
-        ENDIF()
+			add_subdirectory(${CMAKE_SOURCE_DIR}/tests/meta)
+		ENDIF()
 	ENDIF()
 ENDIF()
 
 IF(EXISTS ${CMAKE_SOURCE_DIR}/examples)
 	IF(ENABLE_TESTING AND NOT BUILD_EXAMPLES)
-	    message(STATUS "Tests require (disabled) examples, enabling.")
```

Comment:
these whitespace changes are polluting the diff a bit. Would you mind removing them?

Reply:
well, next time!

Reply:
Hi, sorry about that!